client/state/users/suggestions/reducer.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	USER_SUGGESTIONS_RECEIVE,
+	USER_SUGGESTIONS_REQUEST,
+	USER_SUGGESTIONS_REQUEST_SUCCESS,
+	USER_SUGGESTIONS_REQUEST_FAILURE,
+} from 'state/action-types';
+
+/**
+ * Returns the updated requests state after an action has been dispatched. The
+ * state maps site ID keys to whether a request for user suggestions is in progress.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function requesting( state = {}, action ) {
+	switch ( action.type ) {
+		case USER_SUGGESTIONS_REQUEST:
+		case USER_SUGGESTIONS_REQUEST_SUCCESS:
+		case USER_SUGGESTIONS_REQUEST_FAILURE:
+			return {
+				...state,
+				[ action.siteId ]: USER_SUGGESTIONS_REQUEST === action.type,
+			};
+	}
+
+	return state;
+}
+
+/**
+ * Returns the updated items state after an action has been dispatched. The
+ * state maps site ID keys to an array of user suggestions.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function items( state = {}, action ) {
```

Comment:
Note that in general, temporary state like fetching status is not interesting to persist across browser sessions, and can in some cases cause our application state to be incorrect. (Think if we persisted that a fetch was in flight, refreshed the page, and then we never attempted to fetch again).

I would recommend checking if the suggestions data would be useful to cache. If so we can create a JSON schema to see if the data shape matches. This allows us to change our data shape in the future, and toss out old cached data. [example schema](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/media-storage/schema.js)